linux/README.md

```diff
@@ -11,11 +11,15 @@ Copy the files from this directory, then run one of the install scripts,
 	bash install_centos6_nginx.sh
 	bash install_centos6_py27_apache24.sh
 	bash install_centos6_py27_nginx.sh
+	bash install_centos6_py27_ius_apache22.sh
+	bash install_centos6_py27_ius_apache24.sh
+	bash install_centos6_py27_ius_nginx.sh
 	bash install_debian8_nginx.sh
```

Comment:
Ah, forgot that CentOS 7 is a bit special and cannot not be installed fully automatic, so might be fine to not list it here explicitly.